docs/source/quick-start.mdx

```diff
@@ -82,9 +77,25 @@ Or if you prefer to work from a Jupyter or Colaboratory notebook, then login wit
 >>> notebook_login()
 ```
 
-2. Enter your User Access Token to authenticate your identity to the Hub.
+<Tip>
+
+You can also provide your token to our functions and methods. This way you don't need to
+store your token anywhere.
+
+</Tip>
+
+<Tip>
+
+Once you are logged in, all requests to the Hub will use your access token by default.
+If you want to disable implicit use of your token, you should set the
+`DISABLE_IMPLICIT_HF_TOKEN` environment variable.
```

Comment:
i'm wondering if we don't have a convention of prefixing our env vars with `HF_`? 

what do the other env vars look like?

Reply:
From a quick scan we have:
- `HUGGINGFACE_CO_STAGING`
- `HF_ENDPOINT`
- `HF_HOME`
- `HUGGINGFACE_HUB_CACHE`
- `HF_HUB_OFFLINE`
- `HF_HUB_DISABLE_PROGRESS_BARS`

There's a striking lack of consensus, but I think preceding it by `HF_` would make sense: `HF_HUB_DISABLE_IMPLICIT_TOKEN` ?

Reply:
`HF_HUB_DISABLE_IMPLICIT_TOKEN` sounds definitely better ! Thanks for raising this and proposing a solution :)